signer/src/testing/context.rs

```diff
@@ -206,6 +211,28 @@ impl<Storage, Bitcoin, Stacks>
     }
 }
 
+/// DKG can be triggered if the last DKG signer set differs from the one in config.
+/// However, we don't want to test this functionality in some of our tests, so
+/// this function makes sure that DKG won't be triggered because of changes in signer set.
+pub fn prevent_dkg_on_changed_signer_set<Storage, Bitcoin, Stacks, Emily>(
```

Comment:
Could this not have been a method on `TestContext` directly (i.e. in the block below)?